daemon/main.go

```diff
@@ -146,6 +147,15 @@ func initConfig() {
 }
 
 func initEnv() {
+	socketDir := path.Dir(socketPath)
+	if err := os.MkdirAll(socketDir, 0700); err != nil {
```

Comment:
`700` was the mode we had before. I just restored the code. I'm fine with `750` though.